Cargo.toml

```diff
@@ -25,6 +25,7 @@ rand = "0"
 phf = "0.7"
 phf_macros = "0.7"
 lazy_static = "0.1.10"
+serde = "*"
```

Comment:
Version number please. Cargo takes it as "anything SemVer-compatible with this".